include/cetl/variable_length_array.hpp

```diff
@@ -2126,14 +2121,22 @@ class VariableLengthArray<bool, Allocator> : protected VariableLengthArrayBase<u
     /// is out of memory this method will still return the maximum number of elements that could
     /// be stored if the allocator had enough memory, however, it will always return the maximum
     /// size passed into the constructor if that value is less than the allocator's max_size.
+    /// Note that the underlying size_ and capacity_ of VariableLengthArrayBase are in terms
+    /// of bytes and then converted to bits in size() and capacity().  However the max_size_max_
```

Comment:
please rename `max_size_max_` to `max_size_max_bits_`

Reply:
It's in the base class.  However it is undesirable for it to have different semantic meanings in the subclasses though.  I'll make a max_size_max_bits() function similar to size_bits() and capacity_bits().  That should make it slightly clearer.